tests/e2e-openshift/red-metrics/chainsaw-test.yaml

```diff
@@ -0,0 +1,59 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/kyverno/chainsaw/main/.schemas/json/test-chainsaw-v1alpha1.json
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  creationTimestamp: null
+  name: red-metrics
+spec:
+  steps:
+  - name: step-00
+    try:
+    - apply:
+        file: 00-install-storage.yaml
+    - assert:
+        file: 00-assert.yaml
+  - name: step-01
+    try:
+    - apply:
+        file: 01-install-workload-monitoring.yaml
+    - script:
+        content: ./check_user_workload_monitoring.sh
+  - name: step-02
+    try:
+    - apply:
+        file: 02-install-otel-collector.yaml
+    - assert:
+        file: 02-assert.yaml
+  - name: step-03
+    try:
+    - apply:
+        file: 03-install-tempo.yaml
+    - assert:
+        file: 03-assert.yaml
+    - script:
+        content: /bin/sh -c "kubectl get --namespace $NAMESPACE tempo redmetrics -o
+          jsonpath='{.status.conditions[?(@.type==\"Ready\")].status}' | grep True"
+  - name: step-04
+    try:
+    - apply:
+        file: 04-install-hotrod.yaml
+    - assert:
+        file: 04-assert.yaml
+  - name: step-05
+    try:
+    - apply:
+        file: 05-install-generate-traces.yaml
+    - assert:
+        file: 05-assert.yaml
+  - name: step-06
+    try:
+    - apply:
+        file: 06-intall-assert-job.yaml
```

Comment:
- apply:
-        file: 06-intall-assert-job.yaml
+        file: 06-install-assert-job.yaml

Reply:
👀

Reply:
Fixed.
We have a `lint` command, i wonder if it would catch that 🤔